server/conf/docker/docker-compose-dev.all.debug.yml

```
@@ -3,7 +3,7 @@
 ---
 services:
   cms:
-    image: taccwma/core-cms:latest
+    image: taccwma/core-cms:feat-WC-274-serve-footer-markup # e46d45d5
```

The new feature should now be in `latest` / [v4.31.0](https://github.com/TACC/Core-CMS/releases/tag/v4.31.0).
```suggestion
    image: taccwma/core-cms:latest
```